src/main/java/org/sonarlint/intellij/finding/LiveFinding.java

```diff
@@ -64,6 +67,7 @@ protected LiveFinding(Issue issue, PsiFile psiFile, @Nullable RangeMarker range,
     this.uid = UID_GEN.getAndIncrement();
     this.context = context;
     this.quickFixes = quickFixes;
+    this.ruleDescriptionContextKey = issue.getRuleDescriptionContextKey().orElse(null);
```

Comment:
Is this something that I removed by mistake at some point ? Or it was not supported ?

Reply:
I don't think it was supported. We were passing null as a contextKey for non taint issues. But since I was trying to make the rule description panel simpler, I did this change to avoid having two cases.